webservices/tasks/refresh.py

```diff
@@ -19,7 +19,6 @@ def refresh():
             manage.update_functions()
             manage.update_aggregates()
             manage.refresh_itemized()
-            manage.update_itemized('e')
             manage.update_schemas()
```

Comment:
And this removed as well, but should be replaced by `manage.refresh_materialized()`, right?